rosetta/rosetta/projects/vit/scripts/multiprocess_pretrain_pgle.sh

```diff
@@ -0,0 +1,108 @@
+#! /bin/bash
+
+# Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+
+T5X_DIR=${PWD}
+
+# Arguments
+SIZE=$1          # # Model size (small, base)
```

Comment:
I'd recommend that we use the `getopt` pattern to give this script proper arguments and print usage when the input does not contain expected information. This will make the script easier to use and make the command line that runs the script more readable.

An example can be found [here](https://github.com/NVIDIA/JAX-Toolbox/blob/5b70916c96339a4b0627fe483985a4a9eca4a7ed/.github/container/build-jax.sh#L32) and [here](https://github.com/NVIDIA/JAX-Toolbox/blob/5b70916c96339a4b0627fe483985a4a9eca4a7ed/.github/container/build-jax.sh#L72).